inst/help/mlclassificationneuralnetwork.md

```diff
@@ -0,0 +1,85 @@
+Neural Network Classification
+==========================
+
+Feedforward neural networks are predictive algorithms inspired by the biological neural networks that constitute brains. A neuron (node) that receives a signal then processes it and can send signals to neurons connected to it. The signal at a node is a real number, and the output of each node is computed by sending the signal trough the activation function. The number of nodes in the network is intrinsincly linked to model complexity, as high numbers increase the flexibility of the model.
+
+### Assumptions
+- The target is a nominal or ordinal variable. 
+- The predictor variables consist of continuous variables.
+
+### Input 
+-------
+#### Assignment Box 
+- Target: In this box, the variable that needs to be predicted should be entered. 
+- Predictors: In this box the variables that provide information about the target variable should be entered. 
+
+#### Tables  
+- Confusion matrix: Displays a table that shows the observed classes against the predicted classes. Used to assess model accuracy.
+- Class proportions: Displays a table that shows the proportions of each class in the data set, training (and validaton), and test set.
+- Evaluation metrics: Shows commonly used classification evaluation metrics like precision, recall, the F1-score, support and AUC (area under the ROC curve).
+- Network weights: Shows the connections in the neural network together with their weights.
+
+#### Plots
+- Data split: Shows how the data is split into training (and validation), and test set.
+- Classification accuracy: Plots the number of nearest neighbors against the classification accuracy of the model. Accuracy is assessed for the training (and validation) set.
+- ROC curves: Displays ROC curves for each class predicted against all other classes.
+- Andrews curves: Is a way to visualize structure in high-dimensional data. Lines that cluster are observations that are more alike. 
```

Comment:
The "Is a way to" construction breaks with the style where "plotName: verb" is used instead.